tests/cross/src/test/scala/tests/pc/CompletionMatchSuite.scala

```diff
@@ -103,6 +103,33 @@ object CompletionMatchSuite extends BaseCompletionSuite {
     filter = _.contains("exhaustive")
   )
 
+  checkEdit(
+    "exhaustive-sorting",
```

Comment:
Can we add another cross test case for 3rd party defined symbols? Cross tests have the ability to index library dependencies, see `indexScalaLibrary()`